modules/markup/markdown/math/inline_renderer.go

```diff
@@ -21,7 +21,11 @@ func NewInlineRenderer() renderer.NodeRenderer {
 
 func (r *InlineRenderer) renderInline(w util.BufWriter, source []byte, n ast.Node, entering bool) (ast.WalkStatus, error) {
 	if entering {
-		_, _ = w.WriteString(`<code class="language-math is-loading">`)
+		if _, ok := n.(*InlineBlock); ok {
+			_, _ = w.WriteString(`<pre class="code-block is-loading"><code class="chroma language-math display">`)
+		} else {
+			_, _ = w.WriteString(`<code class="language-math is-loading">`)
```

Comment:
It could be `<code class="language-math is-loading {{some-extra-class-for-inline-or-not}}">`

Reply:
> It could be `<code class="language-math is-loading {{some-extra-class-for-inline-or-not}}">`

`is-loading` is not a common `code` attribute in this case, and the class order can't be changed. If we need to change the order, we might need to update the rule in sanitizer_default.go.

Reply:
My intuition is that single `<code>` is good enough and more maintainable/consistent, while I won't block the two tags.

Reply:
I think the only reason for two tags is semantical correctness.

Reply:
I used two tags to keep the same style as the block render for the `$$` block, but one `code` tag also works.

I have updated it to the one `code` tag version.